src/main/java/org/testng/internal/TestInvoker.java

```diff
@@ -703,13 +703,21 @@ private void runConfigMethods(TestMethodArguments arguments,
 
   public ITestResult registerSkippedTestResult(
       ITestNGMethod testMethod, long start, Throwable throwable) {
```

Comment:
You can implement it into a default method from the interface

Reply:
Fixed